src/fia/fia/fia_dependencies.rs

```diff
@@ -0,0 +1,32 @@
+use crate::prelude::*;
+
+pub(super) struct FiaDependencies<ID, Path, Product>
+where
+    ID: Hash,
+    Path: HasDerivationPath,
+    Product: HasHDPublicKey,
+{
+    pub(super) drivers: Vec<Box<dyn UseFactorSourceDriver<ID, Path, Product>>>,
```

Comment:
one driver for `UseDeviceFactorSource`, one driver for `UseLedgerFactorSource` etc